sample_project/videos/jwaspE_nectar-open-close_control.metadata.yaml

```diff
@@ -16,3 +16,8 @@ Hardware_description: Reference 'nectar-open-close.pdf' on ceph (in \zoo\raw\Lon
 Software_description: Reference 'nectar-open-close.pdf' on ceph (in \zoo\raw\LondonZoo\Experiment-protocols\Jewel-wasp_Ampulex-compressa\)
   or Google Drive (most up-to-date version if they are different - in Experiment Protocols\jewel-wasp_Ampulex-compressa\)
 Further_description: '-'
+Events:
+  experiment_start: 0
+  box_1_open: 25123
+  box_1_closed: 50246
+  experiment_end: 75369
```

Comment:
I'm OK with inferring start/end of video to be the `experiment_start` and `experiment_end` but I'd avoid baking this in. Because .. er... experimental science is hard.

Kind of a ballooning comment: but we may need to support videos completely straddling experiments in weird ways. Memory cards can run out and need hot swapping, batteries run out and... real life.

Reply:
Probably we **do** always want _an_ `experiment_start` and an `experiment_end` to be registered somewhere.

Reply:
On second thought you are right Sam. Defining custom `experiment_start` and `experiment_end` is a nice way for researchers to "crop" videos in time, when things go wrong